prisma/migrations/20240408161437_init/migration.sql

```diff
@@ -0,0 +1,58 @@
+-- CreateEnum
+CREATE TYPE "StatusType" AS ENUM ('backlog', 'unstarted', 'started', 'completed', 'cancelled');
+
+-- CreateEnum
+CREATE TYPE "ViewMode" AS ENUM ('board', 'list');
+
+-- CreateEnum
+CREATE TYPE "FilterType" AS ENUM ('person', 'team', 'client', 'all');
+
+-- CreateEnum
+CREATE TYPE "AssigneeType" AS ENUM ('iu', 'client', 'company');
+
+-- CreateTable
+CREATE TABLE "StatusSettings" (
+    "id" UUID NOT NULL DEFAULT gen_random_uuid(),
+    "type" "StatusType" NOT NULL,
+    "name" TEXT NOT NULL,
+    "key" TEXT NOT NULL,
+    "color" TEXT,
+
+    CONSTRAINT "StatusSettings_pkey" PRIMARY KEY ("id")
+);
+
+-- CreateTable
+CREATE TABLE "Tasks" (
+    "id" UUID NOT NULL DEFAULT gen_random_uuid(),
+    "assigneeId" TEXT,
+    "assigneeType" "AssigneeType",
+    "title" TEXT NOT NULL,
+    "body" TEXT,
+    "statusSettingId" UUID NOT NULL,
+    "createdBy" TEXT NOT NULL,
+
+    CONSTRAINT "Tasks_pkey" PRIMARY KEY ("id")
+);
+
+-- CreateTable
+CREATE TABLE "ViewSettings" (
```

Comment:
you can leave out all of this stuff for ViewSettings for now. Let's keep our implementation simple and just focus on the basic functionality needed for m1.